Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -102,3 +102,38 @@ the pressure drop of the heating coil to zero, and
 to lump the pressure drop of the heating coil into the pressure drop model
 of the cooling coil.
 This often reduces the size of the system of nonlinear equations.
+
+Working process
+---------------
+
+Followings are the working process of contributing.
+
+1. create issue to describe the potential problem or what change should be done
+   to the library. This will generate a issue number (``##``). A good
+   practice is that each issue should address only one problem.
+2. create development branch,
```

Comment:
- Need to precise where the branch is created: locally or remotely. If locally, need to clone or pull before.
- I would use "feature" branch to comply with git official naming, see [https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)